react/features/base/icons/svg/index.ts

```diff
@@ -87,6 +87,7 @@ export { default as IconSubtitles } from './subtitles.svg';
 export { default as IconTileView } from './tile-view.svg';
 export { default as IconTrash } from './trash.svg';
 export { default as IconUserDeleted } from './user-deleted.svg';
+export { default as IconUserGroups } from './user-groups.svg';
```

Comment:
Please don't add back icons that we previously removed. Can't we use IconUser instead of adding IconUserGroups?

Reply:
reverted this add and stick to the IconUsers.